packages/ckeditor5-theme-lark/theme/ckeditor5-html-embed/htmlembed.css

```diff
@@ -118,3 +122,17 @@
 	}
 }
 
+.ck-content[dir="rtl"] .ck-widget.raw-html-embed {
```

Comment:
These are not content styles and they should not start with `.ck-content`. Both the label and the buttons wrapper are not in the editor output data and the styles should not be [listed in the guide](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/content-styles.html#the-full-list-of-content-styles).

I think this should be done using `@mixin ck-dir rtl { ... }`.